### DISCUSSION TITLE
Not able to use Slack Integration in Github Actions

### DISCUSSION THREAD

**Discussion by amitsinghsfdc** [community/community#37771] [Enterprise]
Hi SEMs,

I am working on a GitHub workflow where the requirement is to post the Job status into Slack and I found "https://github.com/slackapi/slack-github-action" actions that seem to be good for our requirement.

However, the workflow is not even starting and giving the below error

**_slackapi/slack-github-action@v1.23.0 is not allowed to be used in devforce/github-actions-tbc-repo. Actions in this workflow must be: within a repository that belongs to your Enterprise account, created by GitHub, or matching the following: goreleaser/goreleaser-action@v2*, hashicorp/setup-terraform@v1._**

Here is my workflow code that I am using

```
- name: slack notification
          uses: slackapi/slack-github-action@v1.23.0
          with:
            # The following message update step does not accept a channel name.
            # Setting a channel ID here for consistency is highly recommended.
            channel-id: "CHANNEL_ID"
            payload: |
              {
                "text": "Deployment started (In Progress)",
                "attachments": [
                  {
                    "pretext": "Deployment started",
                    "color": "dbab09",
                    "fields": [
                      {
                        "title": "Status",
                        "short": true,
                        "value": "In Progress"
                      }
                    ]
                  }
                ]
              }
          env:
            SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN}}
```

I have tried other slack actions as well but getting a similar kind of error.

Any help would be really appreciated.

Thanks & Regards,
Amit Singh

**Comment by ossanna16** [comment]
Hi there @amitsinghsfdc and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
